modules/bucksenseBidAdapter.js

```diff
@@ -0,0 +1,171 @@
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { BANNER } from 'src/mediaTypes';
+
+const WHOIS = 'BKSHBID-008';
+const BIDDER_CODE = 'bucksense';
+const URL = 'https://prebid.bksn.se:445/prebid';
+var bksdebug = false;
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {object} bid The bid to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+  */
+  isBidRequestValid: function (bid) {
+    if (isNaN(bid)) {
```

Comment:
The `bid` param is an object at this stage of the code, so this if statement check will always return `true` and cause the bidRequest for your bidder to be considered invalid.

Can you review this logic and adjust it accordingly?

Reply:
fixed with new Push